packages/shared/src/components/cards/CollectionCard/CollectionCard.spec.tsx

```diff
@@ -86,6 +86,6 @@ it('should display the `collection` pill in the header', async () => {
 it('should show options button on hover when in laptop size', async () => {
   renderComponent();
   const header = await screen.findByLabelText('Options');
-  expect(header).toHaveClass('flex');
+  expect(header).toHaveClass('inline-flex');
```

Comment:
Why was this change needed, I don't see the relevant change

Reply:
ButtonV2 is using `inline-flex` instead of `flex` className.

As a result, `OptionButton` is also using `inline-flex`. Hence this test was failing. Honestly I don't understand why we test for the className there in the first place, but this was a quick fix...

Reply:
Oh why did we change it from `flex` to `inline-flex`?